apps/sync/js/sync-engine/syncengine.js

```diff
@@ -106,6 +106,7 @@ var SyncEngine = (function() {
             payloadEnc => {
           return {
             id: record.id,
+            last_modified: record.last_modified,
```

Comment:
Actually, we could improve https://github.com/mozilla-b2g/gaia/blob/master/apps/sync/test/unit/sync-engine/syncengine_test.js#L238 so that it tests for `expect(Object.keys(se._collections.history.pushData[0]).to.deep.equal(['id, payload'])`